core/ustring.cpp

```diff
@@ -734,7 +734,7 @@ Vector<String> String::split_spaces() const {
 	return ret;
 }
 
-Vector<String> String::split(const String &p_splitter, bool p_allow_empty) const {
+Vector<String> String::split(const String &p_splitter, bool p_allow_empty, int maxsplit) const {
```

Comment:
To follow the Godot conventions, the argument should be named `p_maxsplit` (`p_` for parameter).
I'm thinking it might be better to make `p_maxsplit` the second argument too. I'm not sure what's the purpose of `p_allow_empty`, but it will be less used that the maxsplit.

Reply:
Keep it 3rd for now to avoid having to change the uses of `p_allow_empty` throughout the code base. I'm trying to figure out why this option exists in the first place (maybe it should be unexposed from scripting bindings).